gcp_variant_transforms/libs/vcf_field_conflict_resolver_test.py

```diff
@@ -34,40 +37,164 @@ def _field_count(self, symbol):
     # symbol = {'A', 'G', 'R'}.
     return vcf.parser.field_counts[symbol]
 
-  def test_resolving_field_definition_conflict_in_type(self):
+  def test_resolving_schema_conflict_type(self):
+    test_data_configs = [
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': 1, 'resolved_field_data': True},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': ['1', '2'], 'resolved_field_data': [True, True]},
+        {'schema_type': TableFieldConstants.TYPE_INTEGER,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': '1', 'resolved_field_data': 1},
+        {'schema_type': TableFieldConstants.TYPE_INTEGER,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': ['1', '2'], 'resolved_field_data': [1, 2]},
+        {'schema_type': TableFieldConstants.TYPE_FLOAT,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': 1, 'resolved_field_data': float(1)},
+        {'schema_type': TableFieldConstants.TYPE_FLOAT,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [1, 2], 'resolved_field_data': [float(1), float(2)]},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': 1, 'resolved_field_data': '1'},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [1, 2], 'resolved_field_data': ['1', '2']},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': '', 'resolved_field_data': False},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': ['', ''], 'resolved_field_data': [False, False]},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': [], 'resolved_field_data': None},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [], 'resolved_field_data': []},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': [], 'resolved_field_data': False},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [], 'resolved_field_data': []},
+    ]
+
+    for config in test_data_configs:
```

Comment:
Consider creating a private method for this (e.g. `_run_resolve_schema_conflict_tests`) and pass in the test data config to it as it's the same code in all 3 methods.

Reply:
Done!